packages/containers/package.json

```diff
@@ -89,7 +89,7 @@
     "react-storybook-cmf": "^0.2.0",
     "react-stub-context": "^0.7.0",
     "react-test-renderer": "^15.6.2",
-    "react-virtualized": "9.10.1",
+    "react-virtualized": "9.15.0",
```

Comment:
Should not you update to the latest version: 9.16.0 released ago 7 days :)

Reply:
No breaking change in changelog, let's do it

Reply:
In fact it's already on 9.16.1 in version.js ... I just forgot to run it :/

Reply:
Done